Лабораторная работа №1/Разделение игроков на две команды.py

```diff
@@ -0,0 +1,7 @@
+list_players = ["Маша", "Петя", "Саша", "Оля", "Кирилл", "Коля"]
+count_players = len(list_players)
+middle_index = count_players // 2
```

Comment:
Аналогично и в этой задаче, вы можете поместить вместо переменной `count_players` сразу метод определения количества элементов в списке
```suggestion
middle_index = len(list_players) // 2
```